src/backend/distributed/sql/udfs/alter_role_if_exists/latest.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION pg_catalog.alter_role_if_exists(
+    role_name text,
+    utility_query text)
+    RETURNS void
```

Comment:
the function actually returns bool, and it might be useful. So can you update here to return bool as well?